pdpserver/server/service.go

```diff
@@ -118,9 +118,7 @@ func (s *Server) rawValidate(p *pdp.PolicyStorage, c *pdp.LocalContentStorage, i
 		return pb.Response_INDETERMINATE, []error{err}, nil
 	}
 
-	if s.opts.logger.Level >= log.DebugLevel {
```

Comment:
I don't understand the point of this check. Isn't Debug the highest verbosity level anyways?
Also logrus should now support this check. Is this for a previous bug on logrus' side?

Reply:
Please revert the change. It's an optimization. WithFields function takes insane amount of time and with no the check it's called each time even if log is higher than DebugLevel. Logrus checks log level on later stage inside Debug function which is (as I said) called after WithFields.

Reply:
Is the time issue on our side or on logrus' side? If it is on our side because describing the context is a lot of work, perhaps it's a good target for the upcoming detailed control feature.

Reply:
See https://github.com/sirupsen/logrus/issues/125